docs/installation/bootstrap.rst

```diff
@@ -47,6 +47,7 @@ Configuration
       networks:
         controlPlane:
           cidr: <CIDR-notation>
+          ingressIP: <IP-for-ingress>
```

Comment:
I would suggest we use two levels for this:

```yaml
networks:
  controlPlane:
    ingress:
      ip: <IP-for-ingress-controller-svc>
```

So we can add `ingress.host` (or `ingress.fqdn` :shrug:) later, and use it to sign certs and also in OIDC URLs (issuer URL, redirect URLs...).

Reply:
Also, maybe the field should state `ingressController`? Since it's not "one Ingress object" in particular... I don't know, maybe it's obvious given it's part of the `networks` section :shrug:

Reply:
To me it's not really linked to the controller itself, it's like the IP you use to reach ("Ingress into") Control Plane stuffs.

For the extra layer, I'm not 100% convinced that we should do it now, I see the benefits, and what you suggest may make sense but it's not needed for the moment (and maybe it will never be, I don't know) so it makes the configuration file a bit more complex for ... no reason (at least for the moment) and changing the configuration file layout because we support new features later make sense at the end.

Honestly no strong opinion here, I can change it

Reply:
> it's not really linked to the controller itself, it's like the IP you use to reach ("Ingress into") Control Plane

OK, that makes sense indeed.

> it makes the configuration file a bit more complex for ... no reason (at least for the moment) and changing the configuration file layout because we support new features later make sense at the end.

IMHO, the added complexity today (need to break a word into two, add a newline character) is ridiculous compared to the amount of code + doc changes we need when changing the config file schema.